BibBuddy/app/src/main/res/layout/fragment_author_form.xml

```diff
@@ -34,12 +34,19 @@
                 >
 
                 <!-- title -->
+                <TextView
+                        android:text="@string/book_form_notice"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:id="@+id/textView2"
```

Comment:
```suggestion
                        android:id="@+id/info_mandatory"
```

Reply:
Did adjust & also added an ID in the book form xml.